lib/diagnostic.js

```diff
@@ -38,19 +52,32 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: 'Dalek Blue',
+  species: 'Alien',
+  breed: 'Dalek',
+  vocalization: 'Exterminate!',
+  vocalize: function() {
+    // code goes in here
+  }
+};
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
-const Pet = function() {
-  /* Your answer here */
+const Pet = function(name, species, breed, vocalization) {
+  this.name: name;
+  this.species: species;
+  this.breed: breed;
+  this.vocalization: vocalization;
 };
 
```

Comment:
This is right in ES6, most people right it like this however. I had to check the syntax myself.

const Pet = function(name, species, breed, vocalization) {
 this.name = name;
 this.species = species;
 this.breed = breed;
 this.vocalization = vocalization;
  };